modules/host/host.go

```diff
@@ -46,29 +51,54 @@ type Host struct {
 	tpool  modules.TransactionPool
 	wallet modules.Wallet
 
+	// Host Context.
+	blockHeight  types.BlockHeight
+	recentChange modules.ConsensusChangeID
+	netAddress   modules.NetAddress
+	publicKey    types.SiaPublicKey
+	secretKey    crypto.SecretKey
+	settings     modules.HostSettings
+
 	// File management.
-	fileCounter         int
+	fileCounter         int64
+	spaceRemaining      int64
 	obligationsByID     map[types.FileContractID]*contractObligation
 	obligationsByHeight map[types.BlockHeight][]*contractObligation
-	profit              types.Currency
-	spaceRemaining      int64
 
-	// Persistent settings.
-	blockHeight types.BlockHeight
-	netAddr     modules.NetAddress
-	secretKey   crypto.SecretKey
-	publicKey   types.SiaPublicKey
-	modules.HostSettings
+	// Statistics
+	profit types.Currency
 
 	// Utilities.
 	listener   net.Listener
-	log        *log.Logger
+	log        *persist.Logger
```

Comment:
switched to a persist.Logger to save some code, and to get consistency with the miner module